resources/osx/preinstall.erb

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
 
-# If we appear to be in an upgrade unload services.
+# Cleanup previous operations
+if [ -e "/tmp/.<%= @name %>.upgrade" ]
+  rm -rf "/tmp/.<%= @name %>.upgrade"
+fi
 
+# If we appear to be in an upgrade unload services.
 foundpkg=`/usr/sbin/pkgutil --volume "$3" --pkgs=<%= @identifier-%>.<%= @name -%>`
 oldbom="/usr/share/doc/<%= @name %>/bill-of-materials"
 
 if [ -n "$foundpkg" ]; then
+  mkdir -p "/tmp/.<%= @name %>.upgrade"
+  chmod 0700 "/tmp/.<%= @name %>.upgrade"
```

Comment:
Yeah, I see that; my fault for skimming on the weekend and not **reading** it for reals.